src/components/Button/Button.tsx

```diff
@@ -95,6 +96,7 @@ export const Button: FC<ButtonProps> = ({
     icon,
     children,
     onClick,
+    hugWidth = false,
```

Comment:
It should be `true` by default as we don't want button to be full width by default